pom.xml

```diff
@@ -122,11 +122,11 @@
         <graalvm.version>22.3.0</graalvm.version>
         <gson.version>2.9.0</gson.version>
         <guice.version>4.0</guice.version>
-        <hadoop.version>3.3.0</hadoop.version>
+        <hadoop.version>3.3.1</hadoop.version>
         <ozone.version>1.3.0</ozone.version>
         <hamcrest.version>2.2</hamcrest.version>
         <hbase.version>2.4.6</hbase.version>
-        <hive.version>3.1.2</hive.version>
+        <hive.version>4.0.0</hive.version>
```

Comment:
should this be a string pointing to our backport? something like "3.1.3-with-backport"?

Reply:
I think that we need to follow these steps for that to work

* Update Hive3 version in all pom files and rename it to `3.1.3-with-backport`
* Build Hive3, this will store version `3.1.3-with-backport` under the `~/.m2/repository/`
* Update all Ranger pom files to point to hive version `3.1.3-with-backport`
* Build Ranger and resolve all code conflicts
  * We have to basically downgrade from 4.0.0 to 3.1.3

We just have to make sure that Hive is always built locally, before Ranger is built.

I have a branch `hive4-version-hack` where hive version is set to 3.1.5 in all pom files. I built locally and version 3.1.5 exists under every package unde the `.m2` path. So its doable if you want me to do that. I think it should be good to do it in another Ranger branch and keep this one for working with Hive4. 

```
❯ ls -lah ~/.m2/repository/org/apache/hive/hive-common 
total 24K
drwxrwxr-x  5 xbis xbis 4,0K Απρ  10 15:17 .
drwxrwxr-x 46 xbis xbis 4,0K Απρ  10 15:27 ..
drwxrwxr-x  2 xbis xbis 4,0K Απρ  10 12:49 3.1.3
drwxrwxr-x  2 xbis xbis 4,0K Απρ  10 15:17 3.1.5
drwxrwxr-x  2 xbis xbis 4,0K Απρ  10 12:34 4.0.0
-rw-rw-r--  1 xbis xbis  337 Απρ  10 15:17 maven-metadata-local.xml
```